src/components/dashboard/DashboardActivity.js

```diff
@@ -62,17 +60,22 @@ const DashboardActivity = ({ image, title, to, activity }) => (
          </Grid.Column>
       </Grid.Row>
 
-      {activity.map((row, i) => (
-         <ListItem key={`d-${i}`} row={row} />
+      {transactions.map((transaction, i) => (
+         <ActionsList
+            key={`a-${i}`}
+            transaction={transaction} 
+            actions={transaction.actions}
+            wide={false}
+         />
       ))}
 
       <Grid.Row>
          <Grid.Column textAlign='left' width={16}>
-            <Link to={to}>
+            <a href={to} target='_blank' rel='noopener noreferrer'>
```

Comment:
I don't think it's desired to open a new window on VIEW ALL click (i.e. I think should be no `target='_blank'`)

BTW, is there any specific difference between `react-router-dom`'s `<Link>` and normal `<a>` link?

Reply:
I think, that Link does not support the external URL. If we want to use Link we can add relative path like `to='/explorer'` and then in Route component add redirection to external URL. Would you like me to implement it?